pkg/dependencies/factory_test.go

```diff
@@ -43,6 +43,7 @@ func TestFactoryBuildWithProvider(t *testing.T) {
 
 	tt.Expect(err).To(BeNil())
 	tt.Expect(deps.Provider).NotTo(BeNil())
+	tt.Expect(deps.DockerClient).To(BeNil(), "it only only builds deps for vsphere")
```

Comment:
```suggestion
	tt.Expect(deps.DockerClient).To(BeNil(), "it only builds deps for vsphere")
```
unless you really wanted to stress the *only* part 😛